sonar-plugin-api/src/main/java/org/sonar/api/config/internal/MapSettings.java

```diff
@@ -36,13 +40,15 @@
 public class MapSettings extends Settings {
 
   private final Map<String, String> props = new HashMap<>();
+  private final ConfigurationBridge configurationBridge;
 
   public MapSettings() {
-    super(new PropertyDefinitions(), new Encryption(null));
+    this(new PropertyDefinitions());
   }
 
   public MapSettings(PropertyDefinitions definitions) {
     super(definitions, new Encryption(null));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "Settings"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)